### ISSUE TITLE
is it possible to add an CRF layer?

### ISSUE THREAD

**Issue by Hansyvea** [mohammadKhalifa/xlm-roberta-ner#4]

**Comment by mukhal** [comment]
Yes, it certainly is possible, although I did not observe significant (if any) performance gain from using CRFs

**Comment by Hansyvea** [comment]
> 
> 
> Yes, it certainly is possible, although I did not observe significant (if any) performance gain from using CRFs

could you please add the CRF part into the code?
